shared/pkg/utils/cyclonedx_helper/cyclonedx_helper.go

```diff
@@ -102,7 +102,11 @@ func GetComponentLicenses(component cdx.Component) []string {
 		// https://github.com/CycloneDX/specification/blob/1.4/schema/bom-1.4.xsd#L1398
 		// https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/
 		if license.License != nil {
-			licenses = append(licenses, license.License.ID)
+			if license.License.ID != "" {
```

Comment:
Maybe worth adding a comment here, license must either have one of ID OR Name specified according to the CDX spec, otherwise its invalid CDX:
https://cyclonedx.org/docs/1.4/json/#tab-pane_components_items_licenses_items_oneOf_i0

Reply:
done